crates/release_plz/src/args/mod.rs

```diff
@@ -71,22 +76,66 @@ fn parse_config(config_path: Option<&Path>) -> anyhow::Result<Config> {
             },
         }
     } else {
-        match first_file_contents([
+        let file_contents = first_file_contents([
             Path::new("release-plz.toml"),
             Path::new(".release-plz.toml"),
-        ])? {
-            Some((config, path)) => (config, path),
-            None => {
-                info!("release-plz config file not found, using default configuration");
-                return Ok(Config::default());
-            }
+        ])?;
+        if let Some(project_manifest) = project_manifest
+            .or(Some(Path::new(CARGO_TOML)))
+            .filter(|v| v.exists())
```

Comment:
if the project manifest exists, you ignore completely the `file_contents`, returning the default configuration on line 100, right? 🤔 

Imagine somebody that doesn't specify the config in the Cargo.toml metadata. Their `release-plz.toml` file is ignored.